### ISSUE TITLE
Pressing ^c while image preview is open fills the page with characters but ^l fixes it

### ISSUE THREAD

**Issue by rnmhdn** [ranger/ranger#1908]
#### Runtime Environment
ranger version: ranger 1.9.3
Python version: 3.8.2 (default, Feb 26 2020, 22:21:03) [GCC 9.2.1 20200130]
Locale: en_US.UTF-8

#### Context
my preview method is ueberzug

#### Steps to reproduce
1. Put your cursor on a file that has image preview
2. press `:`
3. press `^C`

#### Traceback
This is the text that is shown:
```
 File "/usr/lib/python3.8/concurrent/futures/process.py", line 233, in _process_worker                                                                        4.75M sum, 64.7G free  12/12  All
                                                                                          call_item = call_queue.get(block=True)
                                                                                                                                  File "/usr/lib/python3.8/multiprocessing/queues.py", line 96, in get
           with self._rlock:
                            Traceback (most recent call last):
                                                                File "/usr/lib/python3.8/multiprocessing/synchronize.py", line 95, in __enter__
                                                                                                                                                   return self._semlock.__enter__()
                                                                                                                                                                                     File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
                                                                       self.run()
                                                                                 KeyboardInterrupt
                                                                                                    File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
                                                                                                                                                                                     self.run()
                                                                                                                                                                                                File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
                                                                           self._target(*self._args, **self._kwargs)
                                                                                                                      File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
                                                                                                                                                                                                self._target(*self._args, **self._kwargs)
                                            File "/usr/lib/python3.8/concurrent/futures/process.py", line 233, in _process_worker
                                                                                                                                     call_item = call_queue.get(block=True)
                                                                                                                                                                             File "/usr/lib/python3.8/concurrent/futures/process.py", line 233, in _process_worker
                                                                       call_item = call_queue.get(block=True)
                                                                                                               File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
                                                                                                                                                                                                self.run()
             File "/usr/lib/python3.8/multiprocessing/queues.py", line 96, in get
                                                                                     with self._rlock:
                                                                                                        File "/usr/lib/python3.8/multiprocessing/synchronize.py", line 95, in __enter__
                                                                                                                                                                                           return self._semlock.__enter__()
                              File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
                                                                                                        self._target(*self._args, **self._kwargs)
                                                                                                                                                   File "/usr/lib/python3.8/concurrent/futures/process.py", line 233, in _process_worker
                                             call_item = call_queue.get(block=True)
                                                                                   KeyboardInterrupt
                                                                                                      File "/usr/lib/python3.8/multiprocessing/queues.py", line 97, in get
                                                                                                                                                                              res = self._recv_bytes()
         File "/usr/lib/python3.8/multiprocessing/connection.py", line 216, in recv_bytes
                                                                                             buf = self._recv_bytes(maxlength)
                                                                                                                                File "/usr/lib/python3.8/multiprocessing/connection.py", line 414, in _recv_bytes
                      buf = self._recv(4)
                                           File "/usr/lib/python3.8/multiprocessing/connection.py", line 379, in _recv
                                                                                                                          chunk = read(handle, remaining)
                                                                                                                                                         KeyboardInterrupt
```

**Comment by toonn** [comment]
I can't reproduce this with the kitty method. @seebye, could this be specific to the ueberzug preview class?

**Comment by aranmohyeddin** [comment]
When previewing some images I get this:
```
/usr/lib/python3.8/site-packages/PIL/Image.py:988: UserWarning: Palette images with Transparency expressed in bytes should be converted to RGBA images
                                                                                  warnings.warn(
```
But I guess your pull request would also solve this so I'm not opening a new issue.

**Comment by rtng** [comment]
![Zed0tos](https://user-images.githubusercontent.com/67840237/142886025-133e10ed-b828-4f8e-82c1-0d9b053cc035.png)

**Comment by toonn** [comment]
Fixed by merging #1913.
